release/_bom.sh

```diff
@@ -224,58 +224,46 @@ function generate_pom_release_dxp_bom_third_party {
 		-e "w ${pom_file_name}" \
 		"${_RELEASE_TOOL_DIR}/templates/release.dxp.bom.third.party.pom.tpl" > /dev/null
 
-	local property_files=(
-		"${_PROJECTS_DIR}/liferay-portal-ee/lib/development/dependencies.properties"
-		"${_PROJECTS_DIR}/liferay-portal-ee/lib/portal/dependencies.properties"
-	)
+	IFS=$'\n' read -d '' -ra dependencies_properties < "${_PROJECTS_DIR}/liferay-portal-ee/lib/development/dependencies.properties"
+	IFS=$'\n' read -d '' -ra portal_dependencies_properties < "${_PROJECTS_DIR}/liferay-portal-ee/lib/portal/dependencies.properties"
```

Comment:
@lucasmiranda0, use the following isn't enough?
```
IFS=$'\n' read -d '' -ra dependencies_properties << "${_PROJECTS_DIR}/liferay-portal-ee/lib/portal/dependencies.properties"
```
I mean, concatenate the content of `${_PROJECTS_DIR}/liferay-portal-ee/lib/portal/dependencies.properties` into the existing array named `dependencies_properties`

Reply:
I tried that but it didn't work.